scripts/qa_crowbarsetup.sh

```diff
@@ -6041,6 +6041,9 @@ function onadmin_batch
         get_novacontroller
         safely oncontroller magnum_service_setup
     fi
+    if grep -q "barclamp: tempest" ${scenario}; then
+        safely crowbar batch --include tempest --timeout 2400 build ${scenario}
```

Comment:
Why having a lower default timeout here?

Reply:
right, I forgot that those timeouts were removed. thanks